scripts/wrapper/apt-get.py

```diff
@@ -7,7 +7,11 @@
 
 def main(argv=sys.argv[1:]):
     max_tries = 10
-    known_error_strings = ['Failed to fetch', 'Hash Sum mismatch']
+    known_error_strings = [
+        'Failed to fetch',
```

Comment:
Does this run inside a docker with `LC_ALL=C` / `LANG=C` ? If so, that's fine, but if there's a different system locale, this may fail to catch these strings.